azure-function/dotnet/Models/Position.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace GetBusData.Models;
+
+public sealed class Position
+{
+    [JsonProperty("latitude")]
```

Comment:
I don't think this `JsonProperty(...)` is necessary because the function app applies camelCasing by default.